int64 index
int64 id_region
<<<<<<< HEAD
int64 parent
=======
>>>>>>> 5f79198d86948135cef3a03bcd2814573960aa96
#variable para grafo de puntos RRT
ltlplanner/position point
float64 value
